packages/component-base/src/dom-utils.js

```diff
@@ -40,6 +40,20 @@ export function getAncestorRootNodes(node) {
   return result;
 }
 
+/**
+ *
```

Comment:
Please add a description and missing typings.

Reply:
https://github.com/vaadin/web-components/pull/5895